wrangler-proto/src/main/java/io/cdap/wrangler/proto/workspace/v2/Workspace.java

```diff
@@ -159,9 +175,23 @@ public Builder setInsights(JsonObject insights) {
       return this;
     }
 
+    public Builder setNullabilityMap (HashMap<String, UserDefinedAction> nullabilityMap) {
+      this.nullabilityMap = nullabilityMap;
+      return this;
+    }
+
     public Workspace build() {
       return new Workspace(workspaceName, workspaceId, directives, createdTimeMillis, updatedTimeMillis, sampleSpec,
-                           insights);
+                           insights, nullabilityMap);
     }
   }
+
+  /**
+   * UserDefinedAction enum.
+   */
+  public enum UserDefinedAction {
+    SKIP_ROW,
+    SEND_TO_ERROR_COLLECTOR,
```

Comment:
ERROR_COLLECTOR is a pipeline specific concept, this can just be SEND_TO_ERROR

Reply:
But by SEND_TO_ERROR_COLLECTOR, I actually mean the pipeline specific error collector hence the name.